### ISSUE TITLE
Sitemap last mod date is wrong.

### ISSUE THREAD

**Issue by awong-dev** [SPS-By-The-Numbers/transcripts#63]
The last mod date should be the last change to speakerinfo, the page, the metadata inside, or any sort of mass site restucture.

Fix that.

**Comment by kzys** [comment]
Is this still open? I'm happy to help!

**Comment by awong-dev** [comment]
Hi @kzys. Happy to have help and the bug is definitely still open!

For this one, I think there's 3 things that need to happen.

First, we'd need a Realtime Database Trigger on at least /public/[category]/metadata/[videoId] and /public/[category]/v/[videoId] that notices when video id underneath them are updated

Second, we'd need to create a /private/[category]/stats/[videoId] object with a last_updated key that we populate on the trigger above

Third, we make the sitemap reference /private/[category]/stats/[videoId]

If you're up for that kind of work, let me know.

**Comment by kzys** [comment]
Yes! Let me setup a dev environment. Do you already have some database triggers I can take a look?

**Comment by awong-dev** [comment]
@kzys  Awesome! There are no such triggers yet. We would need

https://firebase.google.com/docs/functions/database-events?gen=2nd#node.js_1

FYI, the setup doc are likely out of date (sorry...this project has suffered from too few participants).  It might be a good idea to join #dev on this slack so we can chat faster:

https://join.slack.com/t/spsbythenumbers/shared_invite/zt-18zrdd0dh-HloXUNn4zgKR0ML3NeCFKg

There won't be much history as it's been inactive for a few months and slack removed it...but then we can debug faster than over github issues.

**Comment by kzys** [comment]
Sorry. Swamped by other stuff, but now I'm going to spend more time on open source projects. Just joined the Slack.
